server/pulp/server/db/querysets.py

```diff
@@ -67,6 +67,19 @@ def update(self, *args, **kwargs):
                 # if post_save() is not defined for this particular document, that's ok
                 pass
 
+    def get_or_404(self, **kwargs):
```

Comment:
Is the _or_404 redundant to the:
```
:raise pulp_exceptions.MissingResource
```
Not opposed but method names don't typically contain information about raised exceptions.